src/app/donation-start/donation-start-container/donation-start-container.component.html

```diff
@@ -7,8 +7,28 @@
   </div>
   <div class="c-form-container">
     <div class="b-primary-column">
+      <div class="c-header">
+        <h2 class="c-header__to b-grey b-light">Donation to</h2>
+        <h3 class="b-rh-2 b-bold">{{ campaign.charity.name }}</h3>
+        <h2 class="c-header__to b-grey b-light">For</h2>
+        <h1 class="b-rh-2 b-bold">{{ campaign.title }}</h1>
+      </div>
+
+      <app-donation-start-login
+        [logout]="this.logout"
+        [personId]="this.personId"
+        [email]="this.loggedInEmailAddress"
+        [personIsLoginReady]="personIsLoginReady"
+        [loadAuthedPersonInfo]="loadAuthedPersonInfo"
+        [creditPenceToUse]="this.donationStartForm && this.donationStartForm.creditPenceToUse || 0"
+        [campaign]="campaign"
+        [canLogin]="this.canLogin">
+      </app-donation-start-login>
+
       <app-donation-start-primary
+        #donation_start_form
```

Comment:
not sure if this ID is required for the @ViewChild or if we could select it more simply by the component name.

Reply:
I'm not an expert but think I've only seen it done this way or with an Angular class that defines the component. I think we'd probably want to look into the performance of looking it up in a DOM-based way.